.zenodo.json

```diff
@@ -0,0 +1,130 @@
+            "name": "Stuart Mumford",
+            "orcid": "0000-0003-4217-4642"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Steven Christe",
+            "orcid": "0000-0001-6127-795X"
+        },
+        {
+            "affiliation": "Aperio Software",
+            "name": "Nabil Freij"
+        },
+        {
+            "affiliation": "@google",
+            "name": "Florian Mayer"
+        },
+        {
+            "affiliation": "National Institutes of Health",
+            "name": "Keith Hughitt"
+        },
+        {
+            "affiliation": "1 - NASA Goddard Space Flight Center. 2 - Catholic University of America",
+            "name": "Daniel F. Ryan"
+        },
+        {
+            "name": "Simon Liedtke"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Albert Y. Shih"
+        },
+        {
+            "affiliation": "University College London",
+            "name": "David Pérez-Suárez",
+            "orcid": "0000-0003-0784-6909"
+        },
+        {
+            "name": "Pritish Chakraborty"
+        },
+        {
+            "affiliation": "IIT Indore",
+            "name": "Vishnunarayan K I"
+        },
+        {
+            "name": "Andrew R. Inglis"
+        },
+        {
+            "affiliation": "IIIT Hyderabad",
+            "name": "Punyaslok Pattnaik"
+        },
+        {
+            "affiliation": "DIRAC Institute, UW",
+            "name": "Brigitta M. Sipocz",
+            "orcid": "0000-0002-3713-6337"
+        },
+        {
+            "name": "Rishabh Sharma"
+        },
+        {
+            "name": "Drew Leonard"
+        },
+        {
+            "affiliation": "Virginia Tech",
+            "name": "Russell J. Hewett",
+            "orcid": "0000-0001-8944-4705"
+        },
+        {
+            "name": "Alex-Ian-Hamilton"
+        },
+        {
+            "affiliation": "Imperial College London",
+            "name": "David Stansby",
+            "orcid": "0000-0002-1365-1908"
+        },
+        {
+            "affiliation": "International Insititute of Information Technology Bhubaneswar",
+            "name": "Asish Panda"
+        },
+        {
+            "name": "Matt Earnshaw"
+        },
+        {
+            "affiliation": "@kossiitkgp",
+            "name": "Nitin Choudhary"
+        },
+        {
+            "name": "Ankit Kumar"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Laura Hayes"
+        },
+        {
+            "affiliation": "@sunpy  @fossasia @apache",
+            "name": "Prateek Chanda"
+        },
+        {
+            "name": "Md Akramul Haque"
+        },
+        {
+            "name": "Sudarshan Konge"
+        },
+        {
+            "name": "mdmueller"
+        },
+        {
+            "affiliation": "NASA/GSFC",
+            "name": "Michael Kirk"
```

Comment:
@MSKirk ORCID? is the affiliation correct?

Reply:
ORCID: 0000-0001-9874-1429
affiliation: 1 - NASA Goddard Space Flight Center. 2 - Catholic University of America